src/test/goldens/polymer/lib/legacy/legacy-element-mixin.d.ts

```diff
@@ -19,602 +19,608 @@
-     * @param node Node to target.  Defaults to `this`.
-     */
-    toggleClass(name: string, bool?: boolean, node?: Element|null): any;
-
-    /**
-     * Cross-platform helper for setting an element's CSS `transform` property.
-     *
-     * @param transformText Transform setting.
-     * @param node Element to apply the transform to.
-     * Defaults to `this`
-     */
-    transform(transformText: string, node?: Element|null): void;
-
-    /**
-     * Cross-platform helper for setting an element's CSS `translate3d`
-     * property.
-     *
-     * @param x X offset.
-     * @param y Y offset.
-     * @param z Z offset.
-     * @param node Element to apply the transform to.
-     * Defaults to `this`.
-     */
-    translate3d(x: number, y: number, z: number, node?: Element|null): void;
-
-    /**
-     * Removes an item from an array, if it exists.
-     *
-     * If the array is specified by path, a change notification is
-     * generated, so that observers, data bindings and computed
-     * properties watching that path can update.
-     *
-     * If the array is passed directly, **no change
-     * notification is generated**.
-     *
-     * @param arrayOrPath Path to array from which to remove the item
-     *   (or the array itself).
-     * @param item Item to remove.
-     * @returns Array containing item removed.
-     */
-    arrayDelete(arrayOrPath: string|Array<number|string>, item: any): any[]|null;
-
-    /**
-     * Facades `console.log`/`warn`/`error` as override point.
-     *
-     * @param level One of 'log', 'warn', 'error'
-     * @param args Array of strings or objects to log
-     */
-    _logger(level: string, args: any[]|null): any;
-
-    /**
-     * Facades `console.log` as an override point.
-     *
-     * @param args Array of strings or objects to log
-     */
-    _log(...args: any[]): any;
-
-    /**
-     * Facades `console.warn` as an override point.
-     *
-     * @param args Array of strings or objects to log
-     */
-    _warn(...args: any[]): any;
-
-    /**
-     * Facades `console.error` as an override point.
-     *
-     * @param args Array of strings or objects to log
-     */
-    _error(...args: any[]): any;
-
-    /**
-     * Formats a message using the element type an a method name.
-     *
-     * @param methodName Method name to associate with message
-     * @param args Array of strings or objects to log
-     * @returns Array with formatting information for `console`
-     *   logging.
-     */
-    _logf(methodName: string, ...args: any[]): any[]|null;
+  function LegacyElementMixin<T extends new (...args: any[]) => {}>(base: T): T & LegacyElementMixin.Constructor & Polymer.ElementMixin.Constructor & Polymer.GestureEventListeners.Constructor;
+
+  namespace LegacyElementMixin {
+
+    interface Constructor {
+      new(...args: any[]): Interface;
+    }
+
+    interface Interface {
+      isAttached: boolean;
+      _debouncers: any;
```

Comment:
Oh ugh. Missed this one in Polymer I guess... Will need to update my PR for that.